vllm/entrypoints/openai/tool_parsers/xlam_tool_parser.py

```diff
@@ -1,10 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
 # ruff: noqa
 import json
-import re
 from collections.abc import Sequence
 from typing import Any, Dict, List, Optional, Union
 
+import regex as re
+
```

Comment:
(this is needed due to some linting issue on main currently, unrelated to this PR)

Reply:
Fixed on main